### ISSUE TITLE
Train our denoise dataset

### ISSUE THREAD

**Issue by AINewDemo** [swz30/MPRNet#20]
Hello, I am using your published network structure (MPRNet) to retrain a new denoising dataset. Why doesn't the loss function drop? Are there other settings?

**Comment by swz30** [comment]
Hi @AINewDemo 

Hyperparameter settings are provided in the [training.yml](https://github.com/swz30/MPRNet/blob/main/Denoising/training.yml) file. 

Maybe try using a low initial learning rate.
